src/app.js

```diff
@@ -157,22 +156,25 @@ class App {
 
     // center panel, resizable
     self._view.sidepanel = yo`
-      <div id="side-panel" class=${css.sidepanel}>
+      <div id="side-panel" style="min-width: 320px;" class=${css.sidepanel}>
```

Comment:
Set the min-width in CSS. Don't set style inline except if you're modifing it with Javascript.

Reply:
this is done in purpose, I had to do this so I can use in Panel-resize.